core-processor/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     api project(":aop")
     api libs.asm.tree
     api libs.bundles.asm
+    api 'com.github.javaparser:javaparser-symbol-solver-core:3.24.7'
```

Comment:
we are ok adding this extra dependency because core-processor will be a `compileOnly` dependency right?

Reply:
correct

Reply:
I think it should be in libs.versions.toml

Reply:
it should be but as per my comment I didn't want to conflict with @melix's PR so left it like that so we can fix it later